main_test.go

```diff
@@ -64,6 +65,10 @@ func TestComponent(t *testing.T) {
 			Options:              &opts,
 		}.Run()
 
+		fmt.Println("=================================")
```

Comment:
Is this intentional? The code coverage should already be displayed when you run it from the command line.